Makefile

```diff
@@ -1,8 +1,8 @@
-all     :; DAPP_BUILD_OPTIMIZE="1" SOLC_FLAGS="--optimize --optimize-runs=1" \
+all     :; DAPP_BUILD_OPTIMIZE="1" SOLC_FLAGS="--optimize" \
                dapp --use solc:0.6.11 build
 clean   :; dapp clean
 test    :; ./test-dssspell.sh
-deploy  :; DAPP_BUILD_OPTIMIZE="1" SOLC_FLAGS="--optimize --optimize-runs=1" \
+deploy  :; DAPP_BUILD_OPTIMIZE="1" SOLC_FLAGS="--optimize" \
```

Comment:
will this fail if `--optimize-runs=1` is in there?

Reply:
It won't fail, but it will produce inconsistent results with the new format setting runs to 200 and the old setting runs to 1. The new format requires a standard json file to specify the runs as this is no longer specified via the command args. I figured until we add in this standard json we should have consistency between the old and new formats.